docs/conf.py

```diff
@@ -274,8 +286,8 @@ def __getattr__(cls, name):
 #  dir menu entry, description, category)
 texinfo_documents = [
     ('index', 'cookiecutter', u'cookiecutter Documentation',
-     u'Audrey Roy', 'cookiecutter', 'One line description of project.',
-     'Miscellaneous'),
+     u'Audrey Roy and cookiecutter community', 'cookiecutter',
+     'One line description of project.', 'Miscellaneous'),
```

Comment:
While we're here, let's provide this mysterious "one line description"?

Reply:
Any ideas?

Reply:
Will be ``Creates projects from project templates`` in next commit.
Do not like it - propose :)